aptos-move/framework/aptos-framework/sources/transaction_context.spec.move

```diff
@@ -2,7 +2,10 @@ spec aptos_framework::transaction_context {
     spec get_script_hash(): vector<u8> {
         pragma opaque;
         aborts_if false;
+        // property 4: Fetching the script hash of the current entry function should never fail
```

Comment:
Move this comment to the line 4 because the `aborts_if false` spec is also related to this high-level requirement.

Reply:
sure